cpp/CMakeLists.txt

```diff
@@ -401,8 +401,19 @@ if(BUILD_CUML_CPP_LIBRARY)
     src/holtwinters/holtwinters.cu
     src/kmeans/kmeans.cu
     src/knn/knn.cu
-    src/metrics/metrics.cu
+    src/metrics/accuracy_score.cu
+    src/metrics/adjustedRandIndex.cu
+    src/metrics/completenessScore.cu
+    src/metrics/entropy.cu
+    src/metrics/homogeneityScore.cu
+    src/metrics/klDivergence.cu
+    src/metrics/mutualInfoScore.cu
+    src/metrics/pairwiseDistance.cu
+    src/metrics/r2_score.cu
+    src/metrics/randIndex.cu
+    src/metrics/silhouetteScore.cu
```

Comment:
The file names reflect the function they contain in general, right? But this reflects the issue that we have a mixup between underscore and camelcase functions here. This discrepancy is worth revisiting and clearing, if not in this PR in follow up work, so could you open an issue or if you prefer so tackle it in the PR?

Reply:
sure @dantegd , that would be on me 😅 will modify the description of this PR appropriately and clean it up 👍

Reply:
Since that's a fundamentally different modification, can I file a separate follow-up PR?